sharktank/sharktank/evaluate/perplexity_vmfb.py

```diff
@@ -74,7 +74,7 @@ def __init__(
         self.iree_hal_target_backends = iree_hal_target_backends
         self.kv_cache_type = kv_cache_type
         self.activation_dtype = torch.float32
```

Comment:
wasn't activation also float16 by default?

Reply:
Updated to float16.